### PR TITLE
Add Bootstrap 4

### PR DESCRIPTION
**Pull request by TomasVotruba**
### Before

image

### After

image

### CODE REVIEW

File: source/assets/css/style.css [hunk 1]
```diff
@@ -387,3 +389,32 @@ h1:hover .anchor-icon, h2:hover .anchor-icon, h3:hover .anchor-icon, h4:hover .a
...
+
+#mobile-menu .container {
+    padding: 0px;
+    max-width: 100%;
+}
+
+#mobile-menu a {
+    float: left;
+    width: 100%;
+}
```
Comment (enumag) [inline source/assets/css/style.css]: missing newline

File: source/mentori.latte [hunk 2]
```diff
@@ -63,28 +61,32 @@ id: mentori
...
-                    <strong n:tag-if="strlen($mentor['intro_message']) < 140">{$mentor['intro_message']}</strong>
-                </td>
-                <td>
-                    {$mentor['location']}
-                </td>
-                <td><a href="mailto:{$mentor['email']}">{$mentor['email']}</td>
-            </tr>
+            <tbody>
+                {foreach $mentors as $mentor}
+                    {continueIf ($remote && ! $mentor['remote']) || (!$remote && $mentor['remote'])}
```
Comment (enumag) [inline source/mentori.latte]: `$remote xor $mentor['remote']`

File: source/_snippets/head.latte [hunk 3]
```diff
@@ -23,12 +23,12 @@
     <link href="/assets/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
 
     {* Twitter Bootstrap - locally for css classes autocomplete *}
-    <link href="/assets/bootstrap/bootstrap.min.css" rel="stylesheet" type="text/css" />
+    <link href="/assets/bootstrap/bootstrap.min.css?v=0.6" rel="stylesheet" type="text/css" />
```
Comment (grogy) [inline source/_snippets/head.latte]: I don't know why is there version `0.6`?

File: source/assets/css/style.css (hunk 1 above)
Comment (grogy) [inline source/assets/css/style.css]: What about add checker for missing new lines? It is common problem in repositories

File: source/_snippets/head.latte (hunk 3 above)
Comment (enumag) [inline source/_snippets/head.latte]: https://github.com/pehapkari/pehapkari.cz/releases

File: source/assets/css/style.css (hunk 1 above)
Comment (TomasVotruba) [inline source/assets/css/style.css]: Good idea. How would that look like?

Comment (grogy) [inline source/assets/css/style.css]: Best way can be use CSS linter, but for easy use we can use Bash command. Something like

```
test `tail -c 1 source/assets/css/style.css` && echo "missing new line in CSS file" && exit 1
```

Comment (TomasVotruba) [inline source/assets/css/style.css]: I see. What about fixing it right away?

Something like: https://github.com/pehapkari/pehapkari.cz/blob/master/bin/clear_post_trailing_whitespaces.sh

Use here:
https://github.com/pehapkari/pehapkari.cz/blob/d01f67b23044c89034a2b8308a99e12a9cd8f748/composer.json#L38

Could you test and send PR for that please?

Comment (grogy) [inline source/assets/css/style.css]: Done, https://github.com/pehapkari/pehapkari.cz/pull/311 ;-)

File: source/mentori.latte (hunk 2 above)
Comment (TomasVotruba) [inline source/mentori.latte]: I think mixing `or` and  `and` with `!` and `===` might be confusing and lead to more reuse.

Do you knwo how to `xor` in the other syntax?

**Comment by TomasVotruba** [comment]
Thanks for check @enumag and @grogy  
I will finish it in the morning

File: source/assets/css/style.css (hunk 1 above)
Comment (TomasVotruba) [inline source/assets/css/style.css]: I stepped a bit back and realized, how overengeneered my suggestion was. I'm sorry about that.

I wont be so lazy and will fix it by hands :)

File: source/mentori.latte (hunk 2 above)
Comment (TomasVotruba) [inline source/mentori.latte]: I'll keep it for now, to keep it in one form.

File: source/_posts/2016/2016-08-15-podcast-nejenophp-1-o-sebevzdelavani-skoleni-a-mentoringu.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
 ---
+id: 1
```
Comment (TomasVotruba) [inline source/_posts/2016/2016-08-15-podcast-nejenophp-1-o-sebevzdelavani-skoleni-a-mentoringu.md]: ### 1. Create file

```php
# bin/complete-ids.php

<?php declare(strict_types=1);

require_once __DIR__ . '/../vendor/autoload.php';

$finder = Nette\Utils\Finder::find('*.md')->from(__DIR__ . '/../source/_posts');
$files = iterator_to_array($finder->getIterator());

ksort($files);

$i = 1;
foreach ($files as $path => $file) {
    $fileContent = file_get_contents($path);

    $newFileContent = '---' . PHP_EOL . 'id: ' . $i . PHP_EOL . substr($fileContent, 4);

    file_put_contents($path, $newFileContent);
    ++$i;
}
```

### 2. Run in CLI

```bash
php bin/complete-ids.php
```

### 3. Magic :milk_glass:

Comment (TomasVotruba) [inline source/_posts/2016/2016-08-15-podcast-nejenophp-1-o-sebevzdelavani-skoleni-a-mentoringu.md]: In case you'll need it @ikvasnica  :+1:

**Comment by TomasVotruba** [comment]
Merged! :tada: 

Thank you all for checking this PR.

It will be online in 5 minutes: https://pehapkari.cz/mentori/

**Pull request merged**